src/processes/follow-topic-process.js

```diff
@@ -0,0 +1,72 @@
+const StreamChat = require('stream-chat').StreamChat;
+
+const {LogError, UserTopic, CommunityMessageFormat} = require('../databases/models');
+
+const followTopicProcess = async (job, done) => {
+  try {
+    console.info('running job register process ! with id ' + job.id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Avoid using `console.info` for logging in production code. It's better to use a dedicated logging library that can provide different levels of logging (like debug, info, warn, error) and can be configured to output logs to different destinations.

```diff
-    console.info('running job register process ! with id ' + job.id);
+    logger.info('running job register process ! with id ' + job.id);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->